labelbox/schema/foundry.py

```diff
@@ -0,0 +1,160 @@
+from datetime import datetime
+from typing import Any, Dict, Optional, Union
+from pydantic import BaseModel
+from labelbox import exceptions
+from labelbox.schema.identifiables import DataRowIds, GlobalKeys
+from labelbox.schema.task import Task
+from labelbox.utils import _CamelCaseMixin
+
+
+class App(_CamelCaseMixin, BaseModel):
+    id: Optional[str]
+    model_id: str
+    name: str
+    description: str
+    inference_params: Dict[str, Any]
+    class_to_schema_id: Dict[str, str]
+    created_by: Optional[str] = None
+
+    @classmethod
```

Comment:
I understand we have `_NoCoercionMixin` which you wrote yourself :). Curious why we are not using it here

Reply:
The `ResourceNotFoundError` expects this method.